services/user.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str,
+    first_name: str,
+    last_name: str) -> User:
+    user = User.objects.create_user(username=username, password=password)
```

Comment:
Consider using `get_user_model()` to create a user instance instead of directly using `User.objects.create_user()`. This ensures compatibility with custom user models.